### PR TITLE
UPDATED MISSIONS AND VISIONS UI RESPONSIVENESS (PART 1)

### PR DESCRIPTION
**Pull request by mowfaqqa**
First pull request for UI..awaiting review and corrections

### CODE REVIEW

**Comment by abiona01** [comment]
YOU HAVE ONE CONFLICTING FILE, FIX IT, THANK YOU

**Comment by mowfaqqa** [comment]
readjusted a few things on the styling.....awaiting review and feedback

File: client/src/components/header/Header.jsx [hunk 1]
```diff
@@ -40,13 +38,13 @@ export default Header;
...
   display: flex;
   justify-content: center;
   align-items: center;
-  padding: 2.7rem 0.3rem 0.5rem 0.2rem;
-  top: 3rem;
-  margin-bottom: 4.64rem;
+  padding: 1rem 0.2rem 0.5rem 0rem;
+  top: 1rem;
 
   @media only screen and (max-width: 420px) {
```
Comment (abiona01) [inline client/src/components/header/Header.jsx:49]: I think 480 is a better break point, what do you think?

File: client/src/components/organization_mission/missions.style.js [hunk 2]
```diff
@@ -1,83 +1,82 @@
...
+   font-size: 20px;
+   line-height: 20px;
+   top: ${(props) => props.top};
+   left: ${(props) => props.left};
+   color: ${(props) => props.color};
 `;
-
 export const MissionField = styled.p`
-  width: 36vw;
+  width: 40vw;
```
Comment (abiona01) [inline client/src/components/organization_mission/missions.style.js:70]: I'm assuming this is in a div. Why not set the width in % relative to the div

File: client/src/components/organization_vision/org_display_vision/vision.style.js [hunk 3]
```diff
@@ -1,33 +1,37 @@
...
+//     height: 45px;
+    
+//   }
+//   &::placeholder {
+//     font-size: 18px;
+//     font-family: 'Lato';
+//     padding-left: 10px;
+//     color: #999999;
+//   }
+// `;
```
Comment (abiona01) [inline client/src/components/organization_vision/org_display_vision/vision.style.js:34]: Commented out or valid?

**Review by abiona01 — APPROVED** [review]

File: client/src/components/header/Header.jsx (hunk 1 above)
Comment (mowfaqqa) [reply client/src/components/header/Header.jsx:49]: It is but most screens dont reach 480...no worries sha...I'll adjust it

File: client/src/components/organization_vision/org_display_vision/vision.style.js (hunk 3 above)
Comment (mowfaqqa) [reply client/src/components/organization_vision/org_display_vision/vision.style.js:34]: Commented out

File: client/src/components/organization_mission/missions.style.js (hunk 2 above)
Comment (mowfaqqa) [reply client/src/components/organization_mission/missions.style.js:70]: I did buh it was making the input field smaller..so i used Viewport which works just fine

**Pull request merged**